### PR TITLE
[UXIT-1811] Change copy on ecosystem project page

### PR DESCRIPTION
**Pull request by barbaraperic**
## 📝 Description

Copy update on Ecosystem Project Page - [Figma design](https://www.figma.com/design/0pBfQs5tKI6bx5ypPIfVRA/FIL.org-V4-Prototypes?node-id=1231-24805&t=2Ts2islq9XbYIGvb-4)

Adding a [Update an Existing Project CTA is being discussed,](https://filecoinfoundation.slack.com/archives/C04HCHS4RRS/p1733317473404479?thread_ts=1733316305.981589&cid=C04HCHS4RRS) so I'll open a separate PR for that.

### CODE REVIEW

File: src/content/pages/ecosystem-explorer/project-form.md [hunk 1]
```diff
@@ -3,8 +3,8 @@ header:
   title: "Add a Project"
   description:
     [
-      "Let's get started with some basic information about what you're building. Please submit or update your entry to the Filecoin ecosystem page below.",
-      "Want to be a featured project? Share your project page on X and tag @filfoundation to be considered!",
+      "Let's get started with some basic information about what you're building. Please submit your project to the Filecoin ecosystem page below. To update an existing project, send an email to info@fil.org.",
+      "Want to be a featured project? Share your project page on X, tag @filfoundation, and you'll be considered!",
```
Comment (mirhamasala) [inline src/content/pages/ecosystem-explorer/project-form.md:7]: Wouldn't change this - Design probably didn't catch up.

File: src/content/pages/ecosystem-explorer/project-form.md [hunk 2]
```diff
@@ -3,8 +3,8 @@ header:
   title: "Add a Project"
   description:
     [
-      "Let's get started with some basic information about what you're building. Please submit or update your entry to the Filecoin ecosystem page below.",
-      "Want to be a featured project? Share your project page on X and tag @filfoundation to be considered!",
+      "Let's get started with some basic information about what you're building. Please submit your project to the Filecoin ecosystem page below. To update an existing project, send an email to info@fil.org.",
```
Comment (mirhamasala) [inline src/content/pages/ecosystem-explorer/project-form.md:6]: Should go into siteMetadata.ts - Should also be a mailto link.

**Review by mirhamasala — CHANGES_REQUESTED** [review]
Left a couple of comments.

File: src/content/pages/ecosystem-explorer/project-form.md (hunk 2 above)
Comment (barbaraperic) [reply src/content/pages/ecosystem-explorer/project-form.md]: @mirhamasala Do the new changes work?

**Review by mirhamasala — CHANGES_REQUESTED** [review]
@barbaraperic - I wasn't broken up about the implementation. It was providing an incomplete header description and using two different places for the description content.

Let me know what you think of my [CR](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/881/commits/3f3bb598c08007f3eea6cc4dc1c90f23bc569fe4).

Another option we could consider is passing an empty string to the header and handling the description text on the front.

@CharlyMartin - Can you chime in? Thank you!

**Review by CharlyMartin — COMMENTED** [review]
I find it a little weird that Markdown content is in the description front matter while the body of the Markdown file is empty 🙃 But I understand it's a "limitation" of our current setup.

```md
---
description:
  [
    "Let's get started with some basic information about what you're building. Please submit your project to the Filecoin ecosystem page below. To update an existing project, send an email to [info@fil.org](mailto:info@fil.org).",
    "Want to be a featured project? Share your project page on X and tag [@filfoundation](https://twitter.com/filfoundation) to be considered!",
  ]
---
```

> Another option we could consider is passing an empty string to the header and handling the description text on the front.

I think I would find that a bit cleaner yes. Let's roll with that?

I can't think of an alternative solution that doesn't involve reworking Zod schemas and Markdown files.

**Comment by mirhamasala** [comment]
@barbaraperic - Do you agree?

>Another option we could consider is passing an empty string to the header and handling the description text on the front. [>>](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/881#pullrequestreview-2482075847)

>I think I would find that a bit cleaner yes. Let's roll with that? [>>](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/881#pullrequestreview-2482309514)

**Comment by barbaraperic** [comment]
> @barbaraperic - Do you agree?
> 
> > Another option we could consider is passing an empty string to the header and handling the description text on the front. [>>](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/881#pullrequestreview-2482075847)
> 
> > I think I would find that a bit cleaner yes. Let's roll with that? [>>](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/881#pullrequestreview-2482309514)

I like that! I just updated, let me know if it works

File: src/app/ecosystem-explorer/project-form/constants/projectFormDescription.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import { ExternalTextLink } from '@/components/TextLink/ExternalTextLink'
+
+export const PROJECT_FORM_DESCRIPTION = [
+  <>
+    Let's get started with some basic information about what you're building.
+    Please submit your project to the Filecoin ecosystem page below. To update
+    an existing project, send an email to{' '}
+    <ExternalTextLink href={`${FILECOIN_FOUNDATION_URLS.ecosystem.email.href}`}>
+      {`${extractEmailAddress(FILECOIN_FOUNDATION_URLS.ecosystem.email.href)}`}
+    </ExternalTextLink>
```
Comment (mirhamasala) [inline src/app/ecosystem-explorer/project-form/constants/projectFormDescription.tsx:14]:
```suggestion
    <ExternalTextLink href={FILECOIN_FOUNDATION_URLS.ecosystem.email.href}>
      {extractEmailAddress(FILECOIN_FOUNDATION_URLS.ecosystem.email.href}
    </ExternalTextLink>
```

File: src/app/ecosystem-explorer/project-form/constants/projectFormDescription.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  </>,
+  <>
+    Want to be a featured project? Share your project page on X and tag{' '}
+    <ExternalTextLink href={`${FILECOIN_FOUNDATION_URLS.social.twitter.href}`}>
+      {`${FILECOIN_FOUNDATION_URLS.social.twitter.handle}`}
+    </ExternalTextLink>{' '}
```
Comment (mirhamasala) [inline src/app/ecosystem-explorer/project-form/constants/projectFormDescription.tsx:20]:
```suggestion
    <ExternalTextLink href={FILECOIN_FOUNDATION_URLS.social.twitter.href}>
      {FILECOIN_FOUNDATION_URLS.social.twitter.handle}
    </ExternalTextLink>{' '}
```

**Review by mirhamasala — APPROVED** [review]
This is perfect! Thank you for reworking it. 🙌🏼

~ We can remove the interpolation! And then feel free to merge!

**Review by mirhamasala — APPROVED** [review]
🥳❤️

**Pull request merged**